Jetcaster/glancewidget/src/main/res/layout/widget_preview.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you explain why there needs to be an xml file in the compose-samples?

Reply:
For a preview layout so Android between 12 and 15 has accurate widget previews.

Reply:
Can you add a comment to the top of the file?

Reply:
Added